### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by lrapoport-cf**

### CODE REVIEW

**Review by penalosa — APPROVED** [review]
LGTM! It'll probably need revision as we go, but the best way to figure out what's needed is probably to try it!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Fixes # [insert issue number].
```
Comment (rozenmd) [inline .github/pull_request_template.md:1]: I'd probably move this to the bottom of the template

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+What this PR solves / how to test:
+
+Associated docs updates: [insert associated doc update PR(s)]
```
Comment (rozenmd) [inline .github/pull_request_template.md:5]:
```suggestion
Associated docs issues/PR: 
- [insert associated docs issue(s)/PR(s)]
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (lrapoport-cf) [reply .github/pull_request_template.md]: 👍 done :)

**Pull request merged**